content-src/experiments/activity-stream.yaml

```diff
@@ -18,6 +18,9 @@ xpi_url: 'https://moz-activity-streams.s3.amazonaws.com/dist/activity-streams-la
 addon_id: '@activity-streams'
 gradient_start: '#0996F8'
 gradient_stop: '#0675D3'
+min_release:
```

Comment:
This could be a job for #1308 - can leave things out / decide what's required and what's valid in a linting step. We should, like, document what's in these YAML files someday soon too